src/node/board.js

```diff
@@ -1,28 +1,61 @@
-const BoardConstruct = require('./boardConstructor');
-const BoardPrinter = require('./boardPrinter.js');
-const Point = require('./point.js');
+const constants = require('./constants.js');
+const Tile = require('./tile.js');
 
-function Board(size) {
+function Board() {
   const board = {
-    size,
-    matrix: BoardConstruct(size),
-    placePiece(point, symbol) {
-      board.matrix[point.y][point.x].setSymbol(symbol);
+    matrix: BoardConstruct(),
+    placeSymbol: (point, symbol) => {
+      board.matrix[point.x][point.y].setSymbol(symbol);
+      board.matrix[point.x][point.y].setOccupiedToTrue();
     },
-    print: () => {
+    tileIsOccupied: point => {
+      return board.matrix[point.x][point.y].isOccupied;
+    },
+    getSymbol: point => {
+      return board.matrix[point.x][point.y].symbol;
+    },
+    print() {
       BoardPrinter(board);
     },
   };
 
   return board;
 }
 
-//TEST
-const board = Board(3);
-let a = Point(1, 2);
-board.placePiece(a, 'X');
-a = Point(0, 0);
-board.placePiece(a, 'O');
-board.print();
+function BoardConstruct() {
```

Comment:
Since you have a `Board` function which returns a board, having another `BoardConstruct` may confuse the reader, since it appears that there are 2 ways of building a `board`!
How about naming: `function buildBoardTiles() {` ? since it returns a board filled with tiles